xml/System.Web.UI/ScriptManager.xml

```diff
@@ -1260,7 +1260,6 @@
       <MemberSignature Language="C#" Value="public static System.Web.UI.ScriptManager GetCurrent (System.Web.UI.Page page);" />
       <MemberSignature Language="ILAsm" Value=".method public static hidebysig class System.Web.UI.ScriptManager GetCurrent(class System.Web.UI.Page page) cil managed" />
       <MemberSignature Language="DocId" Value="M:System.Web.UI.ScriptManager.GetCurrent(System.Web.UI.Page)" />
-      <MemberSignature Language="VB.NET" Value="Public Shared Function GetCurrent (page As System.Web.UI.Page) As System.Web.UI.ScriptManager" />
```

Comment:
@joelmartinez this change is unexpected. Why mdoc would remove these VB.NET signatures?

Reply:
Never mind. I've noticed those were manually added in #3721